cou/commands.py

```diff
@@ -124,6 +124,19 @@ def get_subcommand_common_opts_parser() -> argparse.ArgumentParser:
         action=argparse.BooleanOptionalAction,
         default=argparse.SUPPRESS,
     )
+    subcommand_common_opts_parser.add_argument(
+        "--archive",
+        help="Archive old database data (nova) before cloud upgrade.\n"
+        "Default to enabling archive.",
+        action=argparse.BooleanOptionalAction,
+        default=argparse.SUPPRESS,
+    )
+    subcommand_common_opts_parser.add_argument(
+        "--archive-batch-size",
+        help="Batch size for nova old database data archiving.\n" "Default is 1000.",
+        type=int,
+        default=1000,
+    )
```

Comment:
Just to verify, this will be added to both `cou plan` and `cou upgrade`? I think that the batch size do not make sense in plan since it will not change anything from plan perpesctive.

Reply:
Yes, but cou has another pattern, e.g.  `--auto-approve` is only on upgrade command. I do not think is an issue, but maybe we can be consistent here.